Tests/DependencyInjection/Fixtures/config/yml/orm_service_single_entity_manager.yml

```diff
@@ -10,7 +10,7 @@ doctrine:
                 memory: true
 
     orm:
-        default_entity_manager: dm2
+        default_entity_manager: default
```

Comment:
The new logic above uncovered this error in the configuration. Looks like we don't check whether the default entity manager even exists 🤷‍♂️